modules/op-configuring-tekton-chains.adoc

```diff
@@ -6,14 +6,22 @@
 [id="configuring-tekton-chains_{context}"]
 = Configuring {tekton-chains}
 
-{tekton-chains} uses a `ConfigMap` object named `chains-config` in the `openshift-pipelines` namespace for configuration.
+The {pipelines-title} Operator installs {tekton-chains} by default. You can configure {tekton-chains} by modifying the `TektonConfig` custom resource; the Operator automatically applies the changes that you make in this custom resource.
 
-To configure {tekton-chains}, use the following example:
+To edit the custom resource, use the following command:
 
-.Example: Configuring {tekton-chains}
-[source,terminal]
+[source,bash]
 ----
-$ oc patch configmap chains-config -n openshift-pipelines -p='{"data":{"artifacts.oci.storage": "", "artifacts.taskrun.format":"tekton", "artifacts.taskrun.storage": "tekton"}}' <1>
+oc edit TektonConfig
```

Comment:
While this command works, IMHO we should update it to `oc edit tektonconfig config` so that it's more aligned with similar commands in our documentation.